corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
I think we'll also want to bundle this with the same account subscription as "bulk operations" like case importer and user importer (which falls in standard and higher). can you work with @nickpell to get that added?

I'm happy to merge now if you want to commit to doing it since it's going to be under a toggle, or wait for you to implement that, whichever you prefer.